pom.xml

```diff
@@ -7,7 +7,7 @@
     <modelVersion>4.0.0</modelVersion>
```

Comment:
please remove the .idea folder above and update the gitignore so this doesn't keep happening